src/Analyzer.Core.BuiltInRuleTests/Tests/Kubernetes_Must_Restrict_Network_Access.json

```diff
@@ -0,0 +1,19 @@
+[
+  {
+    "Template": "Kubernetes",
+    "ReportedFailures": [
+      {
+        "LineNumber": 42,
+        "Description": "No Network restrictions defined"
+      },
+      {
+        "LineNumber": 62,
```

Comment:
Why is 62 flagged, but not 51 or 73? If it because 73 has 75 and 51 is true on 52?

Reply:
The rule requires at least one to be true. Since on line 52 enablePrivateCluster is set to true, this satisfies one condition of the OR statement.